internal/credentials/static.go

```diff
@@ -88,9 +88,7 @@ func (c *Static) Token(ctx context.Context) (token string, err error) {
 		)
 	}
 	defer cc.Close()
-
```

Comment:
Please don't remove  empty lines.

They separate the code into logical blocks for better readability.